contracts/rewards/src/state.rs

```diff
@@ -0,0 +1,383 @@
+use std::collections::HashMap;
+
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{Addr, Storage, Uint256};
+use cw_storage_plus::{Item, Map};
+
+use crate::{error::ContractError, msg::RewardsParams};
+
+#[cw_serde]
+pub struct Config {
+    pub params: RewardsParams,
+}
```

Comment:
why have two different structs that do the same thing? Can we either keep params or config but not both?

Reply:
Have the same question.

Reply:
yeah I think the params should just be stored by themselves as an `Item`. The config in our other contracts is a static set of values set at instantiation that never changes, but the params here are different, since they can be changed.

Also, I think we can combine the params and the last checkpoint into one `Item`